### DISCUSSION TITLE
Highlight how does SeaORM handle the classic N+1 problem

### DISCUSSION THREAD

**Discussion by VaZark** [SeaQL/sea-orm#452] [Ideas]
Since SeaORM is not lazy and can use eager loading to fetch data, I'm assuming the classic N+1 problems that occur in most ORMs are completely sidestepped.

It will be useful for newcomers to learn how to use it if there is an explicit reference to the problem and the solution in the docs.

**Comment by tyt2y3** [comment]
Agreed. We should add some sections to the docs.
